src/presets/azure.ts

```diff
@@ -135,6 +136,14 @@ async function writeRoutes(nitro: Nitro) {
     JSON.stringify(config, null, 2)
   );
   if (!indexFileExists) {
-    await writeFile(resolve(nitro.options.output.publicDir, "index.html"), "");
+    const baseURLSegments = nitro.options.baseURL.split("/").filter(Boolean);
+    const relativePrefix = baseURLSegments.map(() => "..").join("/");
```

Comment:
Could we handle `..` using `new URL()`?

Reply:
you mean, like this?

```js
const baseURLSegments = nitro.options.baseURL.split("/").filter(Boolean);
const relativePrefix = baseURLSegments.map(() => "..").join("/");
const url = new URL(
  joinURL(relativePrefix, "index.html"),
  pathToFileURL(nitro.options.output.publicDir)
);
```

Yes, that works too, but means we convert path -> file URL -> path and I would assume that resolve is just as good at handling the relative prefix.